addons/vehicles/CfgVehicles/Raptor.inc

```diff
@@ -0,0 +1,144 @@
+
+#undef AUTHOR
+#define AUTHOR Demon
+
+class SWS_V_Heli_Raptor_Base : Heli_Attack_02_dynamicLoadout_base_F
+    {
+    ITEM_META(0);
+    displayName = QNAME(Sx-48 Raptor - Base);
+    faction = "SWS_UNSC";
+    side = 1;
+    crew = "B_UNSCAF_Pilot_01";
+    typicalCargo[] = { "B_UNSCAF_Pilot_01" };
+
+    hiddenSelectionsTextures[] = {
+        QPATHTOF(data\raptor\raptor_body1_black_co.paa),
+        QPATHTOF(data\raptor\raptor_body2_black_co.paa)
+    };
+    class TextureSources : TextureSources
+    {
+        class SWS_Basic
```

Comment:
I can go check if needed, but did you confirm the changing of these textures works okay in the Garage in Eden/Zeus? Just making sure.